config/src/config/network_config.rs

```diff
@@ -140,10 +140,10 @@ impl NetworkConfig {
                 self.network_peers.peers.is_empty(),
                 "Validators should not define network_peers"
             );
-        } else if self.is_permissioned {
+        } else if self.enable_remote_authentication {
```

Comment:
please add a check in this function that ensure noise is enabled if enable_remote_authentication is

Reply:
As discussed offline, we'll address these logic changes in a future PR. This PR is just for renaming.